### DISCUSSION TITLE
A Karma or Credit system

### DISCUSSION THREAD

**Discussion by justemu** [outline/outline#2260] [Ideas]
Since ontline is a team knowledge collaboration tool. I think it would be a great idea to provide a karma or credit system.
To be specific: 

- according to the users's contribution to the knowledgebase, e.g. active time consumed in editing or word counts of content creation, to grant the contributors some credits or karma. 
- add a page to honor the top credits contributors. say this week, this month, and total contribution.

This will give the active-sharing team members a sense of honor and , thereby, stimulating creation and sharing of knowledge.

What do you think?

**Comment by tommoor** [comment]
We have had the idea of upvoting or "liking" a document to show thanks to the author, or that it was useful. I think this idea is similar but we have to be careful with such features – time consumed editing or increased word counts are not behaviors that we'd want to incentivize.

**Reply by justemu** [reply]
I got your point. How about adding a page honoring the authors with the most "likes" of the week, of the month, and of the year respectively. It's just some statistics of existing upvotings and might be helpful for better contribution incentives.
